ably.xcworkspace/contents.xcworkspacedata

```diff
@@ -2,6 +2,9 @@
 <Workspace
    version = "1.0">
    <FileRef
+      location = "group:ablySpec/ReadmeExamples.swift">
```

Comment:
This has not much to do with this pull request. Can it be done in a separate one?